target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -424,7 +424,8 @@ ar71xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "2:lan:2" "3:lan:3" "4:lan:4"
 		;;
-	tl-wr842n-v2)
+	tl-wr842n-v2|\
+	tl-wr942n-v1)
 		ucidef_set_interfaces_lan_wan "eth1.1" "eth0"
 		ucidef_add_switch "switch0" \
 			"0@eth1" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
```

Comment:
Here http://fex.net/#227411653165 is my own photos and untouched flash dump right from the box.
For now there is two in power of two different scenarios:
1) We want support USB Leds with both factory and beta u-boots.
2) We want to have working uart.
Both variants needs that GPIO assignments has to be checked and reassigned during boot if nessesary.
But the best is enemy of good. So I think this has to go to TODO list. And, may be some voting and discussing on forum? For now there is nothing to discuss as there is no LEDE support on public.